flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterTableRename.java

```diff
@@ -38,7 +38,15 @@
 
     public SqlAlterTableRename(
             SqlParserPos pos, SqlIdentifier tableName, SqlIdentifier newTableName) {
-        super(pos, tableName);
+        this(pos, false, tableName, newTableName);
+    }
+
+    public SqlAlterTableRename(
+            SqlParserPos pos,
+            boolean ifExists,
```

Comment:
Maybe change to `SqlAlterTableRename(SqlParserPos, SqlIdentifier, SqlIdentifier, boolean)`?